bin/i18n/resources/pegasus/emails.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../../i18n_script_utils'
+
+Dir[File.expand_path('../emails/**/*.rb', __FILE__)].sort.each {|file| require file}
+
+module I18n
+  module Resources
+    module Pegasus
+      module Emails
```

Comment:
Do I need to create a `FILE_NAME` constant here like there is [here](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/i18n/resources/dashboard/base_content.rb#L10)?

Reply:
I don't believe so because, in the sync_in module, you added 
```
LOCALIZABLE_FILE_SUBPATHS = %w[
            hoc_signup_2023_receipt_en.md
          ].freeze
```